### DISCUSSION TITLE
Make DMF NPC's available at major cities when the upcoming location is unreachable due to content lock

### DISCUSSION THREAD

**Discussion by Bolondpali** [chromiecraft/chromiecraft#3434] [Suggestions]
Specifically when it's in Shattrath for example- getting decks is already grindy and turning them in is time gated, people who commit to these items shouldn't be punished to wait 2 months by factors outside their influence. I have seen all the PvP quartermasters have been made available in the major cities, this shouldn't be such a big deal in my opinion.

**Comment by difaxe** [comment]
I fully agree with the above, I don't see an adequate reason to give players an extra time frame when the original game only had 2 real and available DMF venues. // Translated with Google
